salt/search/config/etc-default-gearman-job-server

```diff
@@ -1,10 +0,0 @@
-# This is a configuration file for /etc/init.d/gearman-job-server; it allows
-# you to perform common modifications to the behavior of the gearman-job-server
-# daemon startup without editing the init script (and thus getting prompted by
-# dpkg on upgrades).  We all love dpkg prompts.
-
-# elife 2019-05-28: this is a Ubuntu-only shim for upstart, Gearman doesn't support config from a file
-
-# Parameters to pass to gearmand.
-PARAMS="--libpq-conninfo=postgres://{{ pillar.search.gearman.db.username }}:{{ pillar.search.gearman.db.password }}@127.0.0.1/{{ pillar.search.gearman.db.name }} --queue-type=Postgres --verbose=INFO"
```

Comment:
this is now in https://github.com/elifesciences/search-formula/blob/363d8f8aad3bcc983a237ee3a7b033c87b09fa82/salt/search/config/lib-systemd-system-gearman-job-server.service#L14, for reference

Reply:
yes, it took me a while to unravel this one. This /etc/default/gearman-job-server stuff was sourced by the init.d script that was in turn used by Upstart - none of it official/endorsed by gearman, just the guy who packaged it for Ubuntu. As you say, it's now all contained in the systemd `.service` file